lib/Doctrine/Common/Annotations/DocParser.php

```diff
@@ -940,10 +941,10 @@ private function Constant()
                     break;
 
                 case isset($this->imports[$loweredAlias]):
-                    $found     = true;
                     $className = (false !== $pos)
                         ? $this->imports[$loweredAlias] . substr($className, $pos)
                         : $this->imports[$loweredAlias];
+                    $found     = class_exists($className) || interface_exists($className);
```

Comment:
similar issue with constants. i found no tests for this but think we should handle it the same way? i can also remove this line again if you prefer.

Reply:
Could you rephrase this point? I didn't understand the comment.

Reply:
This addition requires additional tests for:

 * class not found
 * interface not found
 * class found
 * interface found

Reply:
i will revert this change as i am not sure about it. i suspect the same issue to be here for the constants handling as we have for annotations handling, because it also handles namespace alias separately.